solid/solidpython.py

```diff
@@ -528,6 +526,8 @@ def _repr_png_(self):
 
         return png_data
 
+# now that we have the base class defined, we can do a circular import
+from . import objects
```

Comment:
Not much we can do to avoid a circular import here. The alternative would be to put this import in each of the three functions above.